R/trim-osmdata.R

```diff
@@ -1,9 +1,9 @@
 #' trim_osmdata
 #'
-#' Trim an \link{osmdata} object to within a bounding polygon
+#' Trim an `osmdata` object to within a bounding polygon
 #'
-#' @param dat An \link{osmdata} object returned from \link{osmdata_sf} or
-#' \link{osmdata_sp} (DEPRECATED).
+#' @param dat An `osmdata` object returned from [osmdata_sf()] or
```

Comment:
```suggestion
#' @param dat An \link{osmdata} object returned from \link{osmdata_sf} or
```

Reply:
I would leave as is for 2 reasons. First, the `osmdata` doc doesn't apply to `osmdata_sc` objects, so better avoid the link or the even better the `osmdata` word. Second, \link{fun} is equivalent to [fun()] in the .Rd file and is [expanded by roxygen](https://roxygen2.r-lib.org/articles/markdown.html#links) (see .Rd files in the PR).